aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -131,6 +133,78 @@ Tensor& _sigmoid_out_cpu(Tensor& result, const Tensor& self) {
   return result;
 }
 
+// ***** abs *****
```

Comment:
I think you can make use of some of the below macros, no?

If not, I think it'd be cleaner to add, there seems to be a lot of commonality between these functions.